jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/SetupAuthenticationDialog.kt

```diff
@@ -134,8 +136,13 @@ enum class SourceOfEntry {
     }
 }
 
+interface AuthenticationDialog {
+    fun attempts(): Int
```

Comment:
though reading this again, really these should just be included as part of the dialog state instead of being hacked in as an instance property

Reply:
The handlers for this dialog were copied from `SetupAuthenticationDialog` which also does not touch the attempts variable since it uses loginSso under the hood. I need some help to understand how I can update both dialogs to report unsuccessful attempts.